server/portal/apps/system_monitor/unit_test.py

```diff
@@ -66,7 +66,9 @@ def test_system_monitor_when_missing_system(client, settings, requests_mock, sys
     assert system['display_name'] == 'Frontera'
     assert not system['is_operational']
     assert system['load_percentage'] == 0
-    assert system['jobs'] == {'running': 0, 'queued': 0, 'other': 0}
+    assert system['jobs']['running'] == 0
+    assert system['jobs']['queued'] == 0
+    assert system['jobs']['other'] == 0
```

Comment:
This was an unncessary change. After reviewing your comment to my questions, did away with jobs object altogether since TAP response no longer uses it. Did an overhaul using a real TAP system monitor response as a fixture and adjusted tests and view to reflect how system status response actually comes back from server.